<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>896</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog{border-image: url(:/r1.png);}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>29</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnDataStruct">
       <property name="font">
        <font>
         <family>Courier New</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 255);
color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>数据结构</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnComputerComposition">
       <property name="font">
        <font>
         <family>Courier New</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 170, 255);

color: rgb(85, 170, 0);</string>
       </property>
       <property name="text">
        <string>计算机组成原理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOS">
       <property name="font">
        <font>
         <family>Courier New</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0);

color: rgb(170, 85, 255);</string>
       </property>
       <property name="text">
        <string>操作系统</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNET">
       <property name="font">
        <font>
         <family>Courier New</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);
color: rgb(0, 170, 255);</string>
       </property>
       <property name="text">
        <string>计算机网络</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
